.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
A better way to do this (so that we don't have to maintain a list of other people's excluded file types) is to set up a global gitignore file. To do that, follow this guide to [Setting up a global .gitignore file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/) instead, and then remove this file from your branch and try committing. 

It should then silently ignore your .DS_Store file forever, on every repo, and you'll never have to create an individual gitignore file again (until you upgrade your machine and forget you ever did this, anyway).